frontend/src/index.tsx

```diff
@@ -76,11 +77,13 @@ const apolloMiddleware = new ApolloLink((operation, forward) => {
 
 const logoutLink = onError(({ networkError, graphQLErrors }: ErrorResponse) => {
   if (networkError && process.env.REACT_APP_BASE_URL) {
+    // If unauthorized (expired or missing token), remove the access token and reload
     if ("statusCode" in networkError && networkError.statusCode === 401) {
       console.warn("[UNATHORIZED_ACCESS] !!");
-      console.warn("redirect-to:", process.env.REACT_APP_BASE_URL);
       localStorage.removeItem("access_token");
-      window.location.replace(process.env.REACT_APP_BASE_URL);
+      const appUrl = getUrl();
+      console.warn("redirect-to:", appUrl);
+      window.location.replace(appUrl);
```

Comment:
Do you know how often this token expires? I'm wondering if this will affect the experience for users who are just on the app for a long time but shouldn't actually need to log in again.

Reply:
I don't know how often it expires, I'm digging around in the Okta admin console and can't find it. But this should only affect the initial check of the stored token on app load, not when changing pages in the app after the initial load.

Reply:
👍  thanks! I suspect it only expires every 12 hours or so, since that seems to be the cadence at which I'm booted out of the app. And if this only affects initial load, then I think we're good to go!